internal/filter/declinedEvents_test.go

```diff
@@ -0,0 +1,66 @@
+package filter
+
+import (
+	"testing"
+
+	"github.com/inovex/CalendarSync/internal/models"
+	"github.com/stretchr/testify/assert"
+)
+
+// Declined Events should be filtered
+func TestDeclinedEventsFilter(t *testing.T) {
+	sourceEvents := []models.Event{
+		{
+			ICalUID:     "testId",
+			ID:          "testUid",
+			Title:       "test",
+			Description: "bar",
+			AllDay:      true,
+			Accepted:    false,
+		},
+		{
+			ICalUID:     "testId2",
+			ID:          "testUid2",
+			Title:       "Test 2",
+			Description: "bar",
+			AllDay:      false,
```

Comment:
Is the `AllDay` field relevant here?

Reply:
removed, copy and paste failure from the other tests.